scans_test.go

```diff
@@ -161,6 +166,22 @@ func runDockerScan(t *testing.T, testCli *coreTests.JfrogCli, imageName, watchNa
 	}
 }
 
+func runDockerScanWithTokenValidation(t *testing.T, testCli *coreTests.JfrogCli, imageName string, minInactives int) {
+	// Pull image from docker repo
+	imageTag := path.Join(*securityTests.ContainerRegistry, securityTests.DockerVirtualRepo, imageName)
+	dockerPullCommand := container.NewPullCommand(containerUtils.DockerClient)
+	dockerPullCommand.SetCmdParams([]string{"pull", imageTag}).SetImageTag(imageTag).SetRepo(securityTests.DockerVirtualRepo).SetServerDetails(securityTests.XrDetails).SetBuildConfiguration(new(build.BuildConfiguration))
+	if assert.NoError(t, dockerPullCommand.Run()) {
+		defer commonTests.DeleteTestImage(t, imageTag, containerUtils.DockerClient)
+		// Run docker scan on image
+		cmdArgs := []string{"docker", "scan", imageTag, "--validate-secrets", "--server-id=default", "--licenses", "--format=simple-json", "--fail=false", "--min-severity=low", "--fixable-only"}
+		output := testCli.WithoutCredentials().RunCliCmdWithOutput(t, cmdArgs...)
+		if assert.NotEmpty(t, output) {
+			securityTestUtils.VerifyJsonScanTokenValidationResults(t, output, minInactives)
+		}
+	}
+}
+
```

Comment:
similar code to `runDockerScan` can you add an option there?

Reply:
yes